DN.JSonFile.Info.pas

```diff
@@ -219,6 +219,8 @@ procedure TInfoFile.ReadLicenses(const ARoot: TJSONObject);
         LObject := TJSONObject(LValue);
         LLicense.LicenseType := ReadString(LObject, 'type');
         LLicense.LicenseFile := ReadString(LObject, 'file');
+        if not LLicense.LicenseFile.StartsWith('http') then

```

Comment:
I don't see a reason to check for http (why though anyway?). All files should exist in the same repository and the path is expected to be relative.

Reply:
I wasn't sure if say there were multiple licenses, a commercial license may be referenced online . For simplicity, agreed it can be removed. I'll nuke it.

Reply:
I've removed pushed an update for your review.